src/vega.schema.ts

```diff
@@ -329,6 +329,7 @@ export type VgEncodeEntry = {[k in VgEncodeChannel]?: VgValueRef | (VgValueRef &
 export interface VgBinTransform extends BaseBin {
```

Comment:
Let's directly import this type from Vega if we can.

Reply:
Done with https://github.com/vega/vega-lite/pull/5372/commits/deee163256b105761e7bc27d9c038fc722704283. There's still more work to do because `BaseBin` is very similar to `BaseBin` in vega-typings but the latter uses `SignalRefs`. Not sure if there's a straightforward way to import/reuse and strip out `SignalRefs`. Perhaps easier to restructure vega-typings' `BaseBin` to parameterize `SignalRefs` out.